R/getStepwiseDivergence.R

```diff
@@ -22,14 +22,18 @@
 #' samples used to calculate beta diversity
 #' (default: \code{name_timedifference = "time_difference"})
 #' @param assay_name character indicating which assay values are used in
-#' the dissimilarity estimation (default: \code{assay_name = "counts"})
+#' the dissimilarity estimation (default: \code{assay_name = "counts"}).
+#' It could also be used for `reducedDims` entries. Yet, the ones present in 
+#' `assays` are the priority.
```

Comment:
Modify the text as: "This argument can be used to specify data set available through assay or reducedDim slot to be used in divergence calculations. In the case of name conflict (same name in both assay and reducedDim), the assay will have priority."

Reply:
Resolve conversations once done.